pkg/kubelet/kubelet.go

```diff
@@ -2822,6 +2823,12 @@ func (kl *Kubelet) handlePodResourcesResize(pod *v1.Pod) (*v1.Pod, error) {
 		if err := kl.statusManager.SetPodAllocation(pod); err != nil {
 			return nil, err
 		}
+		for i, container := range pod.Spec.Containers {
```

Comment:
another place to support sidecars for https://github.com/kubernetes/kubernetes/pull/128367

@vivzbansal 

I wonder if there is any test that could have catch this.

Reply:
> another place to support sidecars for #128367

@SergeyKanzhelev inside [SetPodAllocation](https://github.com/Nordix/kubernetes/blob/eea8086d309186daad0644ef94d92a721f16fb60/pkg/kubelet/status/status_manager.go#L247) might another place for #128367